Client/src/Utils/Theme/darkTheme.js

```diff
@@ -1,4 +1,6 @@
 import { createTheme } from "@mui/material";
+import { gaugeHeaderCommon } from "./lightTheme"
+export { gaugeHeaderCommon }
```

Comment:
I believe gaugeHader specifics live within gauge component

Reply:
The export is for reuse for the dark theme, so that dark theme does not have to duplicate the same thing